integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -485,56 +483,47 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
 
         let client = clients[client_index];
         let block_header = client.chain.get_block_header(&tip.last_block_hash).unwrap();
-        let shard_layout = client.epoch_manager.get_shard_layout(&tip.epoch_id).unwrap();
```

Comment:
No objection from me on removing this and the print below, but I wonder if anybody was using these print statements while debugging?

Reply:
+1 I think those are good to have.

Reply:
Haven't used them myself. We can add them back locally to debug if needed 👍

Reply:
Will leave on of them, because they duplicate